lxd/resources/network.go

```
@@ -528,43 +525,53 @@ func getNativeBridgeState(name string) *api.NetworkStateBridge {
 }
 
 // Fetch OVS bridge information.
-func getOVSBridgeState(ctx context.Context, ovs *openvswitch.OVS, name string) *api.NetworkStateBridge {
+func getOVSBridgeState(name string) *api.NetworkStateBridge {
+	ovs := openvswitch.NewOVS()
+	isOVSBridge := false
+	if ovs.Installed() {
+		isOVSBridge, _ = ovs.BridgeExists(name)
```

seeing as we're ignored the error here, do we even need to call ovs.Installed()?

---

The only reason I went with this approach was to avoid having to go through each command and letting them fail one by one.

---

ah yeah because Installed only looks for the file being existing. OK lets leave this as is